src/ui-test/tests/debugger.test.ts

```diff
@@ -108,24 +131,84 @@ describe('Camel Debugger tests', function () {
+        } else {
+            skip = true;
+        }
+        // WORKAROUND: https://github.com/redhat-developer/vscode-extension-tester/issues/402
+        // Exchange -> Properties -> from:fromL yaml)
+        const debugView = (await (await new ActivityBar().getViewControl('Run')).openView()) as DebugView;
+        await driver.wait(async function () {
+            try {
+                let variables = await debugView.getContent().getSection('Variables');
+                await variables.openItem('Exchange');
+                const messages = await variables.openItem('Properties');
+                for await (let message of messages) {
+                    if (await message.getAttribute('aria-label') === `from, value ${DEFAULT_PROPERTY}`) {
+                        await driver.actions().doubleClick(message).perform();
+                        await driver.actions().clear();
+                        await driver.actions().sendKeys(TEST_PROPERTY).perform();
+                        return true;
+                    }
+                }
+                return false;
+            } catch (e) {
+                // Extra click to avoid the error: "Element is not clickable at point (x, y)"
+                // Issue is similar to https://issues.redhat.com/browse/FUSETOOLS2-2100
+                if (e instanceof Error && e.name === 'ElementClickInterceptedError') {
+                    await driver.actions().click().perform();  
+                }
+                return false;
+            }
+        }, 240000, undefined, 500);
+
+        // Lack of camel.impl.debugger.BacklogDebugger logs on Exchange property changes
+        // Not possible to verify that value was changed in terminal
+        // Verify that value was changed in Run&Debug variables view
+        let foundMessageWithTestProperty = false;
+        let variables = await debugView.getContent().getSection('Variables');
+        const messages = await variables.openItem('Properties');
+        for await (let message of messages) {
+            if (await message.getAttribute('aria-label') === `from, value ${TEST_PROPERTY}`) {
+                foundMessageWithTestProperty = true;
+            }
+        }
+
+        expect(foundMessageWithTestProperty).to.be.true;
+        await clearTerminal();
+        skip = false;
     });
 
     it('Click on Continue button, and check updated message', async function () {
+        if (skip){ 
+            this.test?.skip(); 
+        } else {
+            skip = true;
+        }
         const debugBar = await DebugToolbar.create();
         await debugBar.continue();
         await waitUntilTerminalHasText(driver, [TEST_MESSAGE]);
-        expect(await (await activateTerminalView()).getText()).to.contain(TEST_MESSAGE);
+        skip = false;
     });
 
-    it('Untoogle breakpoint on log line (14)', async function () {
+    it('Untoggle breakpoint on log line (17)', async function () {
+        if(!breakpointToggled){this.test?.skip();}
```

Comment:
why not to use condition for `it` on line 208?

```
 (breakpointToggled ? it : it.skip)
```

Reply:
Tested

![Screenshot from 2023-08-01 16-26-47](https://github.com/camel-tooling/camel-dap-client-vscode/assets/46084942/25d1bb06-ec4a-47bc-bb16-19c484cea438)

Reply:
I meant more like `(breakpointToggled ? it : it.skip)('Untoggle breakpoint on log line (17)', async function () {`

Reply:
:open_mouth: wow

Reply:
now sonar counts all file conditions :rofl: 
`Refactor this function to reduce its Cognitive Complexity from 55 to the 15 allowed.`

Reply:
sonar was better when did not worked 😆

Reply:
we can consider and change manually in conarcloud as false positive for this condition for `it`

Reply:
Seems that way does not work with changed variables...only with const (as we have for macOS)